### PR TITLE
Empty grid function implementation

### PR DESCRIPTION
**Pull request by meghamittall**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -62,7 +62,13 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    list=[]
```
Comment (Pragya28) [inline battleship.py:65]: Rename list variable as board or grid, so that it is more suitable according to the function.

**Review by Pragya28 — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (Pragya28) [reply battleship.py:65]: @meghamittall make the requested changes, then only I can review it again.

Comment (meghamittall) [reply battleship.py]: @Pragya28 I made changes as you asked me to do. You can please review it now.

**Review by Pragya28 — COMMENTED** [review]
Good work @meghamittall

**Pull request merged**